org.opent2t.sample.lamp.superpopular/com.hue.bulb/js/manifest.xml

```diff
@@ -5,10 +5,14 @@
   <!-- associated schemas -->
   <schemas>
     <schema id="org.opent2t.sample.lamp.superpopular" main="true" type="OCF" />
+    <schema id="opent2t.platform" type="OCF" />
     <schema id="oic.core" type="OCF" />
     <schema id="oic.baseresource" type="OCF" />
+    <schema id="oic.oic-link" type="OCF" />
     <schema id="oic.r.switch.binary" type="OCF" />
     <schema id="oic.r.light.dimming" type="OCF" />
     <schema id="oic.r.colour.rgb" type="OCF" />
+    <schema id="oic.r.colour.chroma" type="OCF" />
```

Comment:
>colour [](start = 22, length = 6)

OK - confirmed on the hue api site that they use the NA variant - 'color' -- let's switch to using that here as well -- [https://www.developers.meethue.com/documentation/core-concepts](https://www.developers.meethue.com/documentation/core-concepts)